test/functional/test-error.js

```diff
@@ -455,7 +455,7 @@ describe('reportErrorToServer', () => {
     expect(data.s).to.be.undefined;
   });
 
-  it('should report experiments', () => {
+  it.should('should report experiments', () => {
```

Comment:
What's this?

Reply:
That was a typo - didn't address because abandoned the CL and just forgot to close. Will do so now.